tests/unit/ui/components/TestShowMoreCard.js

```diff
@@ -1,105 +1,214 @@
 import * as React from 'react';
-import { shallow, mount } from 'enzyme';
+import { shallow } from 'enzyme';
 
 import ShowMoreCard, {
+  extractId,
   ShowMoreCardBase,
   MAX_HEIGHT,
 } from 'ui/components/ShowMoreCard';
 import {
+  applyUIStateChanges,
   createFakeEvent,
   fakeI18n,
   shallowUntilTarget,
 } from 'tests/unit/helpers';
+import { dispatchClientMetadata } from 'tests/unit/amo/helpers';
 
 describe(__filename, () => {
-  const render = (props) => {
+  const getProps = ({ i18n = fakeI18n(), ...props } = {}) => {
+    return {
+      i18n,
+      id: 'showMoreCard',
+      store: dispatchClientMetadata().store,
+      ...props,
+    };
+  };
+
+  function render({ children, ...otherProps } = {}) {
+    const props = getProps(otherProps);
     return shallowUntilTarget(
-      <ShowMoreCard i18n={fakeI18n()} {...props} />,
+      <ShowMoreCard {...props}>{children || 'some info'}</ShowMoreCard>,
```

Comment:
Could you make `'some info'` be the default value of the `children` argument in the function signature, rather than having this `||` here?